packages/backend/src/openapi/webhooks.yaml

```diff
@@ -0,0 +1,275 @@
+openapi: 3.1.0
+info:
```

Comment:
The validator workflow has some warnings
![Screenshot from 2023-03-13 08-52-02](https://user-images.githubusercontent.com/5885521/224721872-a9538c53-5b2a-4f81-ae2e-fa9b9132fad6.png)